protocol/avdl/secret_ui.avdl

```diff
@@ -36,4 +36,22 @@ protocol secretUi {
   GetPassphraseRes getKeybasePassphrase(int sessionID, string username, string retry);
 
   string getPaperKeyPassphrase(int sessionID, string username);
+
+  record SecretStorageFeature {
+    boolean allow;
+    string label;
+  }
+
+  record GUIEntryFeatures {
+    SecretStorageFeature secretStorage;
+  }
+
+  record GUIEntryArg {
+    string windowTitle;
+    string prompt;
```

Comment:
this is based on @cjb doc: https://github.com/keybase/client/blob/8d718a207f1d4e64f1bb6ddfac2325adf0f7da0a/desktop/docs/electron_architecture.md

can just change the prompt during retry?